<!DOCTYPE html>
<html lang="en" dir="ltr">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <!-- Bootstrap v4 Imports -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
    <title>Eric J. Ma's Resume</title>
  </head>

  <body>
    <div class="container">
      <div class="row" id="name-contact-row">
        <div class="col-12" id="name-contact-col">
          <h1 align="center">
            Eric
            
            J.
            
            Ma
          </h1>
          <p align="center">
            <i class="fa fa-envelope" aria-hidden="true"></i>
            <a href="http://shortwhale.com/ericmjl">Email</a>
            |
            <i class="fa fa-phone" aria-hidden="true"></i>
            <a href="tel:857-209-1375">857-209-1375</a>
            |
            <i class="fa fa-linkedin-square" aria-hidden="true"></i>
            <a href="http://www.linkedin.com/in/yourusername">LinkedIn</a>
            |
            <i class="fa fa-github-square" aria-hidden="true"></i>
            <a href="https://github.com/ericmjl">GitHub</a>
            |
            <i class="fa fa-globe" aria-hidden="true"></i>
            <a href="https://ericmjl.github.io/">Personal Website</a>
            |
            <i class="fa fa-map-marker" aria-hidden="true"></i>
            North Quincy, MA
          </p>
        </div>
      </div>
      <br>
      <div class="row" id="skills-title-row">
        <div class="col-12" id="skills-title-col">
          <h2>
            <i class="fa fa-wrench" aria-hidden="true"></i>
            Skills
          </h2>
        </div>
      </div>
      <div class="card-group">
        
        <div class="card">
          <div class="card-body">
            <h5 class="card-title">Languages and Tools</h5>
            <small class="text-muted">
              <p>Programming languages and tools that I use in my day-to-day.</p>
            </small>
            <p class="card-text"> Python, HTML, CSS, bash scripting, git, Illustrator, Indesign, Photoshop</p>
          </div>
        </div>
        
        <div class="card">
          <div class="card-body">
            <h5 class="card-title">Packages</h5>
            <small class="text-muted">
              <p>Specific Python packages that I have proficiency with.</p>
            </small>
            <p class="card-text"> Pandas, PyMC3 (Bayesian statistics), Flask, scikit-learn, scikit-image, Keras, NetworkX, Mesa</p>
          </div>
        </div>
        
        <div class="card">
          <div class="card-body">
            <h5 class="card-title">ML/Stats</h5>
            <small class="text-muted">
              <p>Concepts and ideas that I have mastery over.</p>
            </small>
            <p class="card-text"> network science, machine/deep learning, Bayesian inference, graphical models, agent-based models</p>
          </div>
        </div>
        
        <div class="card">
          <div class="card-body">
            <h5 class="card-title">Life Sciences</h5>
            <small class="text-muted">
              <p>Domain expertise gained during my education.</p>
            </small>
            <p class="card-text"> Microbiology, virology, biochemistry, molecular biology</p>
          </div>
        </div>
        
      </div>
      <br>
      <div class="row" id="experience-row">
        <div class="col-12">
          <h2>
            <i class="fa fa-code" aria-hidden="true"></i>
            Experience
          </h2>
        </div>
        <div class="container" id="experience-details">
          
          <div class="card" id="Novartis Institutes for Biomedical Research (NIBR)-card">
            <div class="card-body">
              <h5 class="card-title">Investigator II, Novartis Institutes for Biomedical Research (NIBR)</h5>
              <small class="text-muted">September 2018-Present, Cambridge MA</small>
              <p class="card-text"><ul>
<li>Investigator in the Scientific Data Analysis (SDA) team reporting to <a href="https://www.linkedin.com/in/holger-hoefling-8418905">Holger Hoefling</a>.</li>
<li>Co-organized and developed teaching material for machine learning &amp; deep learning workshops and seminars internally at NIBR with colleagues <a href="https://www.linkedin.com/in/sivakumar-gowrisankar-b71a016/">Sivakumar Gowrisankar</a>, <a href="https://www.linkedin.com/in/yuan-wang-王元-0b19a822/">Yuan Wang</a>, <a href="https://www.linkedin.com/in/laszlo-urban-4444974/">Laszlo Urban</a>, and <a href="">Sean Xiao</a>.</li>
<li>Developed a massively parallelized engine for training, evaluating, and serving machine learning models on internal assay data, with an emphasis on serving prediction uncertainties with NIBR colleagues <a href="https://www.linkedin.com/in/nikolaus-stiefl-39583b25/">Nikolaus Stiefl</a> and <a href="https://www.linkedin.com/in/gregorigerebtzoff/">Gregori Gerebtzoff</a>.</li>
<li>Developed Bayeisan graph deep learning models for chemical and protein property prediction with <a href="https://www.linkedin.com/in/william-j-godinez/">William J. Godinez</a></li>
<li>Co-mentored <a href="https://www.linkedin.com/in/arkadij-kummer-78b249b9/">Arkadij Kummer</a> with <a href="https://www.linkedin.com/in/richard-lewis-8916891/">Richard Lewis</a> in support of the development of machine learning and deep learning workflows for protein engineering.</li>
</ul></p>
            </div>
          </div>
          <br>
          
          <div class="card" id="Novartis Institutes for Biomedical Research (NIBR)-card">
            <div class="card-body">
              <h5 class="card-title">Investigator I, Novartis Institutes for Biomedical Research (NIBR)</h5>
              <small class="text-muted">September 2017-August 2018, Cambridge MA</small>
              <p class="card-text"><ul>
<li>Investigator in the Scientific Data Analysis (SDA) team reporting to <a href="https://www.linkedin.com/in/borowsky/">Mark Borowsky</a>.</li>
<li>Performed internal consulting projects and expanded the SDA Statistical Learning initiative with colleagues.</li>
<li>Developed parameterized Bayesian agent-based models of internal project portfolio for scenario planning purposes, using <a href="http://docs.pymc.io">PyMC3</a> and <a href="https://github.com/projectmesa/mesa">Mesa</a>.</li>
<li>Assisted in the analysis of high throughput DROSHA cleavage data, with one paper currently in writing.</li>
<li>Assisted in mentoring two interns, <a href="https://www.linkedin.com/in/stacy-meichle/">Stacy Meichle</a> (Computer Aided Drug Discovery with <a href="https://www.linkedin.com/in/clayton-springer-5a48072/">Clayton Springer</a>) and <a href="https://www.linkedin.com/in/flekschas/">Fritz Lekschas</a> (SDA with <a href="https://www.linkedin.com/in/brant-peterson-b390a51/">Brant Peterson</a>).</li>
</ul></p>
            </div>
          </div>
          <br>
          
          <div class="card" id="Insight Data Science-card">
            <div class="card-body">
              <h5 class="card-title">Health Data Science Fellow, Insight Data Science</h5>
              <small class="text-muted">June 2017-August 2017, Boston MA</small>
              <p class="card-text"><ul>
<li>Built <a href="https://fluforecaster.herokuapp.com">Flu Forecaster</a>, a machine learning-powered system that forecasts flu sequences six months out, to better prepare for manufacturing of vaccine strains.</li>
<li>Implemented a variational autoencoder (deep learning model) to learn a continuous representation of 14,455 influenza hemagglutinin protein sequences, and trained a Gaussian process model on the continuous representation to predict new flu sequences.</li>
<li>Developed interactive blog post using Flask and Bootstrap, and deployed to Heroku and GitHub.</li>
<li>Led peer workshops on <a href="https://github.com/ericmjl/minimal-flask-example">web development</a>, deep learning and <a href="https://github.com/ericmjl/pycodestyle-tutorial">code style</a>.</li>
</ul></p>
            </div>
          </div>
          <br>
          
          <div class="card" id="Massachusetts Institute of Technology-card">
            <div class="card-body">
              <h5 class="card-title">ScD Candidate, Massachusetts Institute of Technology</h5>
              <small class="text-muted">August 2011-May 2017, Cambridge MA</small>
              <p class="card-text"><ul>
<li>Developed a scalable, network-based phylogenetic heuristic algorithm for detecting reassortant influenza viruses using 18,632 fully sequenced virus genomes that improved our capacity to detect reassortment events by two orders of magnitude.</li>
<li>The algorithm was used in a lead-author study (published in <a href="http://www.pnas.org/content/113/19/5335.full">PNAS</a>) providing systematic evidence that genome shuffling is important for host switching, and a co-authored study (published in <a href="http://onlinelibrary.wiley.com/doi/10.1111/ele.12629/abstract">Ecology Letters</a>) that showed that reassortment is a strategy for viral gene persistence in wild animal reservoirs.</li>
<li>Contributed <a href="https://github.com/runstadler-lab/Seal-H3N8-Image-Analysis">reproducible data analysis</a> for colleagues through fluorescent image quantification and genomic analysis in studies that refuted binding properties of novel influenza viruses.</li>
<li>Performed <a href="https://github.com/ericmjl/mia-stats/blob/master/sterilization/sterilization.ipynb">Bayesian statistical modelling</a> for colleagues testing the efficacy of phone sterilization tools.</li>
<li>Delivered tutorials and talks on Network Analysis and Bayesian statistical methods at annual Python conferences, including PyCon, SciPy, and PyData.</li>
</ul></p>
            </div>
          </div>
          <br>
          
          <div class="card" id="The University of British Columbia-card">
            <div class="card-body">
              <h5 class="card-title">Undergraduate Researcher, The University of British Columbia</h5>
              <small class="text-muted">June 2006-May 2010, Vancouver BC</small>
              <p class="card-text"><ul>
<li>Investigated the role of T cells in intestinal inflammation and fibrosis in a Salmonella typhimurium-induced model of gut inflammation, leading to First Prize Poster (<a href="https://www.evernote.com/l/AANJS_A9iKZAMLGYqJDduTxiiI2ZNf43zHM">Rising Stars of Research 2008</a>) and Best Talk (Multidisciplinary Undergraduate Research Conference 2009) awards.</li>
<li>Co-founded the <a href="http://2009.igem.org/Team:British_Columbia">first UBC International Genetically Engineered Machines (iGEM) team</a>, where we achieved a Gold status standing in our first year of competition.</li>
<li>Served as a <a href="https://science.ubc.ca/students/spac">Peer Academic Coach</a>, guiding first- and second-year students towards academic success through building good habits on learning strategies and time management.</li>
</ul></p>
            </div>
          </div>
          <br>
          
        </div>
      </div>
      <br>
      <div class="row" id="education-row">
        <div class="col-12" id="education-title-row">
          <h2>
            <i class="fa fa-graduation-cap" aria-hidden="true"></i>
            Education
          </h2>
        </div>
      </div>
      <div class="card-group">
        
        <div class="card">
          <div class="card-body">
            <h5 class="card-title">B.Sc., The University of British Columbia</h5>
            <small>Integrated Sciences, June 2006-May 2010, Cambridge MA</small>
          </div>
        </div>
        
        <div class="card">
          <div class="card-body">
            <h5 class="card-title">Sc.D., Massachusetts Institute of Technology</h5>
            <small>Department of Biological Engineering, August 2011-May 2017, Cambridge MA</small>
          </div>
        </div>
        
      </div>

      <br>
      <div class="row" id="publications-row">
        <div class="col-12" id="publications-title-row">
          <h2><i class="fa fa-bookmark"></i> Publications</h2>
          <ul class="list-group">
            
            <li class="list-group-item">
              pyjanitor: A Cleaner API for Cleaning Data. <i>Proceedings of the 18th Python in Science Conference (SciPy 2019)</i> (2019). <a href="https://pdfs.semanticscholar.org/7b96/42a31f805a7c9584b2e6f6b08576a1e029f7.pdf">[link]</a>
            </li>
            
            <li class="list-group-item">
              Peax: Interactive Visual Pattern Search in Sequential Data Using Unsupervised Deep Representation Learning. <i>BioRXiv</i> (2019). <a href="https://www.biorxiv.org/content/10.1101/597518v2">[link]</a>
            </li>
            
            <li class="list-group-item">
              Long-term colonization dynamics of Enterococcus faecalis in implanted devices in research macaques. <i>Applied and Environmental Microbiology</i> (2018). <a href="https://aem.asm.org/content/early/2018/07/09/AEM.01336-18">[link]</a>
            </li>
            
            <li class="list-group-item">
              Evaluation of 6 Methods for Aerobic Bacterial Sanitization of Smartphones. <i>Journal of the American Association for Laboratory Animal Science</i> (2018). <a href="https://www.ingentaconnect.com/content/aalas/jaalas/2018/00000057/00000001/art00004">[link]</a>
            </li>
            
            <li class="list-group-item">
              Reassortment of influenza A viruses in wild birds in Alaska before H5 clade 2.3. 4.4 outbreaks. <i>Emerging Infectious Diseases</i> (2017). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5367406/">[link]</a>
            </li>
            
            <li class="list-group-item">
              A real-time surveillance dashboard for monitoring viral phenotype from sequence. <i>International Journal of Infectious Diseases</i> (2016). <a href="http://www.ijidonline.com/article/S1201-9712(16)31510-7/abstract">[link]</a>
            </li>
            
            <li class="list-group-item">
              Evidence of seasonality in a host-pathogen system: Influenza across the annual cycle of wild birds. <i>Integrative and Comparative Biology</i> (2016). <a href="http://www.sicb.org/meetings/2016/schedule/abstractdetails.php?id=502">[link]</a>
            </li>
            
            <li class="list-group-item">
              A point mutation in the polymerase protein PB2 allows a reassortant H9N2 influenza isolate of wild-bird origin to replicate in human cells. <i>Infection, Genetics and Evolution</i> (2016). <a href="https://www.sciencedirect.com/science/article/pii/S156713481630137X">[link]</a>
            </li>
            
            <li class="list-group-item">
              New England harbor seal H3N8 influenza virus retains avian-like receptor specificity. <i>Scientific Reports</i> (2016). <a href="https://www.nature.com/articles/srep21428">[link]</a>
            </li>
            
            <li class="list-group-item">
              Ecosystem interactions underlie the spread of avian influenza a viruses with pandemic potential. <i>PLOS Pathogens</i> (2016). <a href="http://onlinelibrary.wiley.com/doi/10.1111/ele.12629/abstract">[link]</a>
            </li>
            
            <li class="list-group-item">
              Transmission of influenza reflects seasonality of wild birds across the annual cycle. <i>Ecology Letters</i> (2016). <a href="http://onlinelibrary.wiley.com/doi/10.1111/ele.12629/abstract">[link]</a>
            </li>
            
            <li class="list-group-item">
              Reticulate evolution is favored in influenza niche switching. <i>PNAS</i> (2016). <a href="http://www.pnas.org/content/113/19/5335.full">[link]</a>
            </li>
            
            <li class="list-group-item">
              Genetic characterization of a rare H12N3 avian influenza virus isolated from a green-winged teal in Japan. <i>Virus Genes</i> (2015). <a href="https://link.springer.com/article/10.1007/s11262-014-1162-9">[link]</a>
            </li>
            
          </ul>
        </div>
      </div>

    </div>
  </body>
  <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
  <!-- FontAwesome embed -->
  <script src="https://use.fontawesome.com/cb9dbe8e41.js"></script>

</html>